stars/default/defaultStarsConfig.json

```diff
@@ -1,93 +1,93 @@
 {
-	"version": 13,
+	"version": 14,
```

Comment:
Did you changed the version number in C++ code?

Reply:
I just recognized this and force pushed to bump the version.

Reply:
Hint: `static const int StarCatalogFormatVersion`

Reply:
I will change that in the morning or you can push the change😅 i have turned off my laptop to go to sleep. I will make a checklist for myself to keep track of what action need to be taken each time I update catalogs and JSON.